code/modules/reagents/reagent_containers/food/drinks.dm

```diff
@@ -263,6 +265,9 @@
 			if(H.species.chem_flags & NO_DRINK)
 				reagents.reaction(get_turf(H), TOUCH)
 				H.visible_message("<span class='warning'>The contents in [src] fall through and splash onto the ground, what a mess!</span>")
+				var/datum/reagents/R = src.reagents
+				var/splashed = R.get_master_reagent_id()
```

Comment:
Doesn't this work differently than the other code? This code takes the master reagent ID but the other code takes ANY reagents.